api/src/pcapi/core/bookings/api.py

```diff
@@ -146,8 +147,16 @@ def get_individual_bookings(user: User) -> list[Booking]:
                 Venue.publicName,
                 Venue.timezone,
             )
+            .joinedload(Venue.offererAddress)
+            .joinedload(OffererAddress.address)
         )
         .options(joinedload(Booking.stock).joinedload(Stock.offer).joinedload(Offer.mediations))
+        .options(
+            joinedload(Booking.stock)
+            .joinedload(Stock.offer)
+            .joinedload(Offer.offererAddress)
```

Comment:
On risque de charger la requête avec beaucoup de colonnes à rapatrier, ne serait-il pas pertinent de limiter ?
Par exemple :
```suggestion
            joinedload(Booking.stock).load_only(Stock.offerId)
            .joinedload(Stock.offer).load_only(Offer.offererAddressId)
            .joinedload(Offer.offererAddress).load_only(OffererAddress.addressId)
```